components/brave_wallet/common/f4_jumble_unittest.cc

```diff
@@ -0,0 +1,266 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/common/f4_jumble.h"
+
+#include <string>
+
+#include "base/containers/span.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace brave_wallet {
+
+// https://github.com/zcash/librustzcash/blob/f4jumble-0.1.0/components/f4jumble/src/test_vectors.rs
```

Comment:
LGTM - thanks for linking to the test_vectors to make sure we're matching their implementation